### DISCUSSION TITLE
How can I set the FLEX_SERVERLESS env var to 1 ?

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#44953] [Q&A]
Fabien in his Twitter say:

> If you want to help us test the new serverless Flex, set the FLEX_SERVERLESS env var to 1. Then, create a new #symfony project or update symfony/flex for existing projects

How can I set the `FLEX_SERVERLESS` env var to 1 ?

And I use Symfony 6.0.2, Is it a need to use this command?

**Comment by helmis123** [comment]
`FLEX_SERVERLESS=1 composer recipes`
